### DISCUSSION TITLE
cefsharp stops working after visual studio 2022 is installed.

### DISCUSSION THREAD

**Discussion by DavidHe-2008** [cefsharp/CefSharp#3969] [General]
On calling code:

  var settings = new CefSettings()

I got exception:
BadImageFormatException: Could not load file or assembly 'CefSharp.Core.Runtime.dll' or one of its dependencies.  is not a valid Win32 application. (Exception from HRESULT: 0x800700C1)

it was good before I install visual studio 2022.

version : 89.0.170

Thanks.

John

**Comment by amaitland** [comment]
> version : 89.0.170

You will need to upgrade to a supported version (currently version 97).

**Reply by amaitland** [reply]
Issue #3854 was fixed some months ago.
